src/server/service/diagnostics/ioload.rs

```diff
@@ -73,7 +85,9 @@ impl IoLoad {
                         .split_whitespace()
                         .map(|w| w.parse().unwrap_or_default())
                         .collect::<Vec<f64>>();
-                    if parts.len() != 11 {
+                    // A not too old Linux kernel supports the first 11 block statistics.
+                    // Others stats are supported by Linux 4.19+, we consider them as optional ones.
+                    if parts.len() < 11 {
```

Comment:
Do you know how clients use this info? Will newly added fields cause trouble, like panic?

Reply:
It is printed out when we select from information_schema's CLUSTER_LOAD table. Extra keys won't cause a problem.